recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - openjdk
+    - openjdk 17
```

Comment:
Out of curiosity: why do we need to pin the JDK?

Reply:
There is a version parsing issue in `bazel` that has an issue with `20-internal` but works with `17.0.1-internal`. We can remove this in future once the parsing issue has been fixed.